src/components/TodosFilter/TodosFilter.tsx

```diff
@@ -0,0 +1,80 @@
+import { useContext } from 'react';
+import classNames from 'classnames';
+
+import { TodosContext } from '../../context/TodosContext';
+
+export const TodosFilter = () => {
+  const {
+    todos, setTodo, filter, setFilter,
+  } = useContext(TodosContext);
+
+  const activeTodos = todos.filter(todo => todo.completed === false);
+  const todosCounter = activeTodos.length;
+  const completedTodos = todos.filter(todo => todo.completed);
+
+  const clearCompleted = () => {
+    setTodo([...activeTodos]);
+  };
+
+  return (
+    <footer className="footer">
```

Comment:
don't render the footer if we don't have the todos
<img width="701" alt="image" src="https://github.com/mate-academy/react_todo-app/assets/70220667/a9d46791-72b9-4d5b-8079-1c188f1c546f">